lib/java/test/org/apache/thrift/test/EqualityTest.java

```diff
@@ -129,10 +129,11 @@ public static void main(String[] args) throws Exception {
 package org.apache.thrift.test;
 
 // Generated code
-import java.nio.ByteBuffer;
 
 import thrift.test.JavaTestHelper;
 
+import java.nio.ByteBuffer;
```

Comment:
This import was moved, but there was a comment above it saying this code was generated. I'm not sure what would have generated this code, because it doesn't specify, but if that comment is still relevant, style changes should not be made to this class. If it isn't relevant, then that comment that was above this import before it moved should be deleted.

Reply:
this is reverted